outbound.go

```diff
@@ -234,6 +234,11 @@ func (call *OutboundCall) Arg3Writer() (ArgWriter, error) {
 	return call.arg3Writer()
 }
 
+// RemotePeer returns the peer info.
```

Comment:
nit: "returns the peer info." to "returns the peer info of the selected connection for this call" maybe?